packages/hash/api/codegen.yml

```diff
@@ -27,6 +27,7 @@ generates:
       #   See https://app.asana.com/0/1201095311341924/1203393349856541/f
       scalars:
         Date: string
+        JSONObject: "@blockprotocol/core#JsonObject"
```

Comment:
Oh, was the `scalar JSONObject` untyped in the base schema before? That's bad :sweat_smile: